packages/common/container-definitions/src/error.ts

```diff
@@ -3,72 +3,32 @@
  * Licensed under the MIT License.
  */
 
-import { ITelemetryProperties } from "@fluidframework/core-interfaces";
+import { IErrorBase } from "@fluidframework/core-interfaces";
 
 /**
- * Different error types the Container may report out to the Host
+ * @deprecated IErrorBase is being deprecated as a public export is moving to "core-interfaces".
  */
-export enum ContainerErrorType {
```

Comment:
I _believe_ our change from enum to const object changes the underlying data representation in a way that is not necessarily backwards compatible for our users (depending on what they're doing with the data). We probably need to continue exporting the original enum representation here (still marked as deprecated).

Hopefully someone with more insight into how and where all this is used can help determine if this is strictly necessary.

Reply:
i think this is a good change, but it will be breaking change. probably best not to couple with the move. need to think if there is a bundle efficient way to have both, thinking maybe the enum can use the consts?

Reply:
I tried to have the enum use the const, but I get the error, so went back to string literals.

> @fluidframework/container-definitions: src/error.ts:35:24 - error TS18033: Only numeric enums can have computed members, but this expression has type '"dataCorruptionError"'. If you do not need exhaustiveness checks, consider using an object literal instead.